consensus/consensus_v2.go

```diff
@@ -792,11 +792,13 @@ func (consensus *Consensus) finalizeCommits() {
 	consensus.consensusTimeout[timeoutConsensus].Start()
 
 	consensus.getLogger().Info().
-		Uint64("blockNum", beforeCatchupNum).
-		Uint64("ViewId", beforeCatchupViewID).
+		Uint64("blockNum", block.NumberU64()).
+		Uint64("ViewId", block.Header().ViewID().Uint64()).
 		Str("blockHash", block.Hash().String()).
 		Int("index", consensus.getIndexOfPubKey(consensus.PubKey)).
 		Msg("HOORAY!!!!!!! CONSENSUS REACHED!!!!!!!")
+	// Print to normal log too
+	utils.GetLogInstance().Info("HOORAY!!!!!!! CONSENSUS REACHED!!!!!!!", "BlockNum", block.NumberU64())
```

Comment:
why? we are phasing out the original logs.  will be no more original logs.

Reply:
We still have those “trie commit to database” stuff on the terminal, which is confusing people. It’s etheruem code so hard to remove. Remember a lots of people asking whether I joined consensus or not, right? . We might as well surface up consensus signal just for people to easily see.

Reply:
Just merged the code so other part get in master for release. Let me know if the log code has to be reverted.